circle.yml

```diff
@@ -37,3 +39,26 @@ test:
     - docker logs addonsserver_worker_1 > $CIRCLE_ARTIFACTS/worker.log
     - cp logs/* $CIRCLE_ARTIFACTS
     - cp geckodriver.log $CIRCLE_ARTIFACTS
+
+deployment:
+  latest:
+    branch: master
+    commands:
+      - docker tag app:build ${DOCKERHUB_REPO}:latest
+      - docker login -e $DOCKERHUB_EMAIL -u $DOCKERHUB_USER -p $DOCKERHUB_PASS
+      - docker push ${DOCKERHUB_REPO}:latest
+
+  releases:
+    # push all tags
+    tag: /.*/
+    commands:
+      - docker tag app:build ${DOCKERHUB_REPO}:${CIRCLE_TAG}
+      - docker login -e $DOCKERHUB_EMAIL -u $DOCKERHUB_USER -p $DOCKERHUB_PASS
+      - docker push ${DOCKERHUB_REPO}:${CIRCLE_TAG}
+
+  branches:
+    branch: push-to-dockerhub-via-circleci
```

Comment:
What's this all about? Are you planning this for testing and delete these lines later?